src/test/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentServiceImplTestPE.java

```diff
@@ -0,0 +1,30 @@
+package edu.cornell.kfs.fp.batch.service.impl;
+
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.core.config.Configurator;
+import org.junit.Before;
+import org.junit.Test;
+
+import edu.cornell.kfs.fp.batch.service.impl.CreateAccountingDocumentServiceImplTestBase.TestCreateAccountingDocumentServiceImpl;
+import edu.cornell.kfs.fp.batch.xml.fixture.AccountingDocumentMapping;
+import edu.cornell.kfs.fp.batch.xml.fixture.AccountingXmlDocumentListWrapperFixture;
+
+public class CreateAccountingDocumentServiceImplTestPE extends CreateAccountingDocumentServiceImplTestBase {
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+        Configurator.setLevel(PreEncumbranceDocumentGenerator.class.getName(), Level.DEBUG);
+        createAccountingDocumentService = new TestCreateAccountingDocumentServiceImpl(buildMockPersonService(),
+                buildAccountingXmlDocumentDownloadAttachmentService(), configurationService,
+                buildMockUniversityDateService(), dateTimeService);
+        createAccountingDocumentService.initializeDocumentGeneratorsFromMappings(AccountingDocumentMapping.PE_DOCUMENT);
+        setupBasicCreateAccountingDocumentServices();
+    }
+    
+    @Test
+    public void testLoadSingleFileWithSinglePEDocument() throws Exception {
```

Comment:
It looks like this is the only PE document unit test case. If it's not practical to add more test cases at this time, then you may want a separate user story to do so.

I think you mentioned in one of our team or standup meetings that the Create Accounting Document unit test setup was fairly cumbersome to work with. Perhaps we also need a follow-up user story to refactor those tests.

Reply:
I created https://jira.cornell.edu/browse/KFSPTS-31641 to improve the cumbersome areas you reminded of

Reply:
I have added more unit tests